data/skeletons/lalr1.cc

```diff
@@ -235,7 +235,29 @@ m4_define([b4_shared_declarations],
     void error (const syntax_error& err);
 
 ]b4_token_constructor_define[
+]b4_parse_error_bmatch([verbose\|detailed\|custom], [[
+    class Context {
```

Comment:
Please, make this `context`.  I followed C++'s "native" style, with lower case class names.